addons/account/models/chart_template.py

```diff
@@ -432,6 +430,11 @@ def _get_default_account(journal_vals, type='debit'):
 
             return default_account
 
+        # As those accounts were originally on the journals, they need to be set here
```

Comment:
@vin-odoo Maybe instead of changing it here, which is a weird, we could do the write above checking all the parent chart templates for everything.